src/sass/user-profile.scss

```diff
@@ -57,7 +57,13 @@
   }
 
   .card {
-    .resume-saved-application-container {
+    p {
+      margin-bottom: 8px;
+    }
+
+    .application-route-container {
```

Comment:
Does this need to be nested under `.card`?

Reply:
No, removed.